buildpack.md

```diff
@@ -606,7 +608,7 @@ The stack ID:
 - MUST only contain numbers, letters, and the charactors `.`, `/`, and `-`.
 - MUST NOT be identical to any other stack ID when using a case-insensitive comparison.
 
-The stack `build-images` and `run-images` are suggested sources of images for platforms that are unaware of the stack ID.
+The stack `build-images` and `run-images` are suggested sources of images for platforms that are unaware of the stack ID. Buildpack authors MUST ensure that these images include all mixins specified in `mixins`.
```

Comment:
So is it correct that the lifecycle will not apply the mixins (either by installing the packages or any other way)? It is entirely the responsibility of the buildpack author?